### PR TITLE
[JFG-902] To get rid of the xml configuration; Unit-tests in progress.

### PR DESCRIPTION
**Pull request by SokolAndrey**

### CODE REVIEW

**Comment by anton-antonenko** [comment]
lgtm

File: chassis/core/src/main/java/com/griddynamics/jagger/user/builders/JTestDescriptionBuilder.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    private String id;
+    private String comment;
+    private String name;
+    private String description;
+    private String version;
+
+    private ScenarioFactory scenarioFactory;
+
+    private List<Provider<InvocationListener<Object, Object, Object>>> listeners;
```
Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/builders/JTestDescriptionBuilder.java]: it is better to use 
`List<Provider<InvocationListener<?, ?, ?>>>`

File: chassis/core/src/main/java/com/griddynamics/jagger/user/builders/JTestDescriptionBuilder.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        ((QueryPoolScenarioFactory) scenarioFactory).setEndpointProvider(queryProvider);
+        return this;
+    }
+
+    public JTestDescriptionBuilder withLoadBalancer(QueryPoolLoadBalancer loadBalancer) {
+        ((QueryPoolScenarioFactory) scenarioFactory).setLoadBalancer(loadBalancer);
+        return this;
+    }
+
+    public JTestDescriptionBuilder withListeners(List<Provider<InvocationListener<Object, Object, Object>>> listeners) {
```
Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/builders/JTestDescriptionBuilder.java]: it is better to use
` List<Provider<InvocationListener<?, ?, ?>>>`

**Comment by SokolAndrey** [comment]
Closed due to misunderstanding of the documentations.

**Pull request closed without merging**